src/releasing/packager.sh

```diff
@@ -126,7 +127,7 @@ if gpg-agent; then
             echo "repository: $r" | sed 's/./=/g'
 
             echo "release tag: "
-            release_tag="$(git tag -l | grep -e "$RELEASE\$")"
+            release_tag="$(git tag -l | grep -e "$VERSION\$")"
```

Comment:
`$VERSION` is something like `"7 (Core)"`. Are you sure that is want you want?

Reply:
Ah yes, overloaded variable name, it's set on [lines 101-104](https://github.com/quattor/release/pull/346/files#diff-e6f5cbc97452a339c45695e32e8e7684fa4d8527da34df6760c8888c8217fdf1R101-R104).

Reply:
I should probably choose a different name.

Reply:
maybe pick QUATTOR_VERSION ?